lib/client-assets.php

```diff
@@ -109,9 +109,16 @@ function gutenberg_register_scripts_and_styles() {
 
 	// Editor Styles.
 	wp_register_style(
+		'gutenberg-mce-skin',
+		includes_url( 'js/tinymce/skins/lightgray/skin.min.css' ),
```

Comment:
If we want our own skin - for an earlier attempt on the freeform block I had converted the TinyMCE lightgray skin to scss (instead of less which TinyMCE uses). I should still have that conversion around if you want it.